### ISSUE TITLE
Cache the feed contents

### ISSUE THREAD

**Issue by K4LCIFER** [thunder-app/thunder#1101]
**Is your feature request related to a problem? Please describe.**
I noticed that when I open a post, and then return back to the feed, the whole feed flickers -- I assume that this is it reloading/re-fetching the feed contents. This causes me to think that the contents are not being cached.

**Describe the solution you'd like**
Cache the feed contents, so that the feed doesn't need to be completely reloaded.

**Describe alternatives you've considered**
None.

**Additional context**
I'm not sure if this is exactly what the problem is, so this feature request is rather malleable.

**Comment by micahmo** [comment]
@K4LCIFER Any chance you could provide a screen recording of what's flickering? Is it mostly the images? If so, we have some plans for extending image cache time.

**Comment by K4LCIFER** [comment]
Yeah it was the images that were flickering. I can still send a video if you would like!

**Comment by micahmo** [comment]
Sure, thank you1

**Comment by K4LCIFER** [comment]
> Sure, thank you!

Hrm, I appear to not be able to reproduce it now. Perhaps it was a symptom of me being on a slower network when I first experienced it?

**Comment by micahmo** [comment]
It comes and goes for me too. I opened #1105, but I have kept it in draft mode because I haven't been able to prove definitively that it helps. 😆
